### PR TITLE
CID-2809 - Update public and internal docs to mark API call for customer to set their own API Key as deprecated

### PR DESCRIPTION
**Pull request by nonameentername**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

File: api-docs/api-reference/methods/users/post-add-credential-to-user-v2.0-users-userid-os-ksadm-credentials.rst [hunk 1]
```diff
@@ -7,6 +7,12 @@ Add credential to user
 
     POST /v2.0/users/{userId}/OS-KSADM/credentials
 
+.. note::
+
+    Add api-key credentials to a user is deprecated.
```
Comment (catlook) [inline api-docs/api-reference/methods/users/post-add-credential-to-user-v2.0-users-userid-os-ksadm-credentials.rst:12]: This **Add API-key credentials to a user** operation is deprecated.

File: api-docs/api-reference/methods/users/post-add-credential-to-user-v2.0-users-userid-os-ksadm-credentials.rst [hunk 2]
```diff
@@ -7,6 +7,12 @@ Add credential to user
...
+    Please use :ref:`Reset API Key<post-reset-api-key-for-user-v2.0>`
```
Comment (catlook) [inline api-docs/api-reference/methods/users/post-add-credential-to-user-v2.0-users-userid-os-ksadm-credentials.rst:13]: Instead, use :ref:`Reset API Key<post-reset-api-key-for-user-v2.0>`.

**Review by catlook — CHANGES_REQUESTED** [review]
Couple of suggestions

**Review by catlook — APPROVED** [review]
👍

**Review by kidster — APPROVED** [review]

**Pull request merged**